src/com/google/cose/Ec2SigningKey.java

```diff
@@ -114,6 +118,74 @@ public ECPublicKey getPublicKey() {
     return (ECPublicKey) this.keyPair.getPublic();
   }
 
+  private static byte[] arrayFromBigNum(BigInteger n, int curveSize) {
+    byte[] rgb = new byte[(curveSize + 7) / 8];
```

Comment:
Nit: Please use a more descriptive name than `rgb`

Reply:
+1 we are not using rg for range notation in our code iirc.

Reply:
renamed to `keyBytes`, done.